src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/processor/ZipVerifiersTest.java

```diff
@@ -82,4 +88,32 @@ public void should_not_verify_invalid_zip_successfully() throws Exception {
         ZipVerifiers.sha256WithRsaVerification(zipStreamWithSig);
     }
 
+    @Test
+    public void should_verify_valid_exela_test_zip_successfully() throws Exception {
```

Comment:
is `should_verify_signed_file_successfully` test still needed?
This one seems a more 'real life' test, where zip files are signed (as opposed to single pdfs)

Reply:
Every kind of file should be signable and working. Using 2 different signatures (1 generated by us and 1 by the other party) gives more guarantees.